apis/logging/v1/kibana_types.go

```diff
@@ -51,29 +54,33 @@ type KibanaStatus struct {
 	Deployment string `json:"deployment"`
 	// +optional
 	ReplicaSets []string `json:"replicaSets,omitempty"`
+	// The status for each of the Kibana pods for the Visualization component
 	// +optional
+	// +operator-sdk:csv:customresourcedefinitions:type=status,displayName="Kibana Status",xDescriptors="urn:alm:descriptor:com.tectonic.ui:podStatuses"
 	Pods PodStateMap `json:"pods,omitempty"`
 	// +optional
 	Conditions map[string]ClusterConditions `json:"clusterCondition,omitempty"`
 }
 
-// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-
-// Kibana is the Schema for the kibanas API
+// +kubebuilder:object:root=true
 // +k8s:openapi-gen=true
 // +kubebuilder:resource:path=kibanas,categories=logging,scope=Namespaced
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="Management State",JSONPath=".spec.managementState",type=string
 // +kubebuilder:printcolumn:name="Replicas",JSONPath=".spec.replicas",type=integer
+// Kibana instance
+// +operator-sdk:csv:customresourcedefinitions:displayName="Kibana",resources={{Deployment,v1}}
```

Comment:
Looking through this list:
https://github.com/openshift/elasticsearch-operator/blob/master/pkg/k8shandler/kibana/reconciler.go#L54

Tells us, that Kibana CR is also creating the following missing resources:
- ConsoleExternalLogLink
- ConsoleLink
- ConfigMap
- Role
- RoleBinding
- Route
- Service
- ServiceAccount

According to OLM documentation these resources are only descriptors to inform users on which resources are owned by a CRD:
```
Resources: Your CRDs will own one or more types of Kubernetes objects. These are listed in the resources section to inform your end-users of the objects they might need to troubleshoot or how to connect to the application, such as the Service or Ingress rule that exposes a database.
```
See https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#owned-crds

I suggest we complete this list then.